src/main/python/hydra/lib/hydrabase.py

```diff
@@ -666,6 +671,66 @@ def get_mesos_slave_ips_attr(self, attr_type, attr_value):
         """
         return self.__mesos.get_slave_ips_from_attribute(attr_type, attr_value)
 
+    def get_all_mesos_slave_attr(self):
```

Comment:
Ok Now i get it. the comment is not clear :)
Yes I can just use self.mesos_cluster in place, but I foresee the need of a function to just get the attr list of all the registered slaves.
Adding an APi to get that doesnt harm in my opinion. We might need to use it later as well.